retnet/modeling_retnet.py

```diff
@@ -589,7 +589,7 @@ def generate(
         bos_token_id=0,
         eos_token_id=0,
         do_sample=False,
-        top_k=1,
+        top_k=0,
```

Comment:
I checked huggingface/transformers repo, and found their implementation has `top_k=50` as default. Still, I agree that setting `top_k=0` in this implementation makes more sense. Thanks!